packages/split-view/test/split-view.test.ts

```diff
@@ -0,0 +1,728 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { fixture, elementUpdated, expect, html } from '@open-wc/testing';
+
+import '../sp-split-view.js';
+import { SplitView } from '..';
+import {
+    arrowDownEvent,
+    arrowLeftEvent,
+    arrowRightEvent,
+    arrowUpEvent,
+    endEvent,
+    homeEvent,
+    pageDownEvent,
+    pageUpEvent,
+    shiftTabEvent,
+} from '../../../test/testing-helpers.js';
+
+describe('SplitView', () => {
+    it('loads default (horizontal) split-view accessibly', async () => {
+        const el = await fixture<SplitView>(
+            html`
+                <sp-split-view primary-default="100">
+                    <div>First panel</div>
+                    <div>Second panel</div>
+                </sp-split-view>
+            `
+        );
+
+        await elementUpdated(el);
+        expect(el).to.be.accessible();
+        expect(el.primaryDefault).to.equal(100);
+        const gripper = el.shadowRoot.querySelector(
+            '#gripper'
+        ) as HTMLDivElement;
+        expect(gripper).not.to.be.accessible();
```

Comment:
What about the gripper is "not accessible"? I'm unsure what this is testing.

Reply:
Oh, actually, I see that it's not testing anything, really. All of the `.accessible()` tests need to be `await`ed or you're not actually getting results.

Reply:
Did you found out if we really need the "await" of if we can follow the linter recommendations to remove it?

Reply:
I've added `await` now to all of this `to.be.accessible();`